api.py

```diff
@@ -157,3 +158,45 @@ class Contains(Resource):
             "locations": locations,
         }
         return json(response, status=200)
+
+@ns_loc_func.route('/overlaps')
+class Contains(Resource):
```

Comment:
I believe this class name conflicts with existing class Contains

Reply:
Yes you're right, thats a copy+paste error. It doesn't cause an error at runtime because the `ns.route()` decorator wraps the class with a new identity. 

But I will push a fix for it anyway.